ibis/expr/operations/relations.py

```diff
@@ -580,6 +580,20 @@ def schema(self):
         return self.table.schema
 
 
+@public
+class Sample(Relation):
+    """Sample performs random sampling of records in a table."""
+
+    table: Relation
+    fraction: Annotated[float, Between(0, 1)]
+    method: Literal["row", "block"] = "row"
```

Comment:
We want to default to "row" since "block" may produce a non-uniform sample (since it exposes information about the block implementation of the backing database).

And since the default values show up in the signature/docs, I'd like to keep the explicit default here if possible.

Reply:
> And since the default values show up in the signature/docs, I'd like to keep the explicit default here if possible.

Hm, not totally following this. How are docs related to ops declaration?

Reply:
Oop, sorry, misreading on my part, I thought this was `expr/relations.py` not `operations/relation.py`. I agree the default isn't needed here.